src/components/notifications/test/NotificationsProvider.test.tsx

```diff
@@ -12,6 +12,7 @@ import { NotificationsProvider } from '../NotificationsProvider';
 import { useNotificationsListener } from '../hooks/useNotificationsListener';
 
 jest.mock('reconnecting-websocket');
+jest.mock('uuid', () => ({ v4: () => '00000000-0000-0000-0000-000000000000' }));
```

Comment:
use constant, cf doc.

```suggestion
import { useNotificationsListener } from '../hooks/useNotificationsListener';
import { NIL as NIL_UUID } from 'uuid';

jest.mock('reconnecting-websocket');
jest.mock('uuid', () => ({ v4: () => NIL_UUID }));
```